vr/agent/publisher.py

```diff
@@ -45,7 +44,7 @@ def main():
     # state change, since the emitted events don't have everything we want.
     rpc = supervisor_utils.getRPCInterface(os.environ)
 
-    rcon = redis.StrictRedis(**parse_redis_url(os.environ['REDIS_URL']))
+    rcon = redis.StrictRedis.from_url(os.environ['REDIS_URL'])
     hostname = os.getenv('HOSTNAME', socket.getfqdn())
     log('proc_publisher starting with hostname %s' % hostname)
     log('connecting to redis at %s' % os.environ['REDIS_URL'])
```

Comment:
If we're going to start soliciting credentials in the URL, we probably don't want to be emitting that value to the logs.

Reply:
Done.